bigchaindb/commands/utils.py

```diff
@@ -14,41 +15,63 @@
 from bigchaindb.version import __version__
 
 
-def start_rethinkdb():
+def start_temp_rethinkdb(port=0, directory=None):
```

Comment:
I used it to spawn and kill RethinkDB to tests the new `Connection` class. Unfortunately Travis wasn't happy about that (I think there is no `rethinkdb` binary when testing, it's just an external service) so I had to rollback and mock up the connection lost behaviour. I can remove this or integrate it with `start_rethinkdb`.